acceptance/bundle/run/jobs/partial_run/databricks.yml

```diff
@@ -0,0 +1,25 @@
+bundle:
+  name: partial_run
+
+resources:
+  jobs:
+    my_job:
+      name: my_job
+      job_clusters:
+        - job_cluster_key: job_cluster
+          new_cluster:
+            num_workers: 2
+            autoscale:
+              min_workers: 1
+              max_workers: 2
+            spark_version: "13.3.x-scala2.12"
+            node_type_id: "Standard_DS3_v2"
```

Comment:
You can omit the job cluster definition and reference for this test.

A task configuration without `job_cluster_key` or `environment_key` is valid as a serverless task.